lib/pushSubscriptions.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Created by Yousef on 6/14/16.
+ */
+
+var util = require('./util')
+    , authenticator = require('./authenticator')
+    , querystring = require('querystring');
+
+var pushSubscriptions = {}
+    , _allowedPostProps = [
+        'object_type'
+        , 'aspect_type'
+        , 'callback_url'
+        , 'verify_token'
+    ];
+    
+pushSubscriptions.endpointBase = 'https://api.strava.com/api/v3/';
+
+pushSubscriptions.post = function(args,done) {
+    var endpoint = 'push_subscriptions';
+    var err = null;
+
+    if(
+       typeof args.object_type === 'undefined' || 
+       typeof args.aspect_type == 'undefined' || 
+       typeof args.callback_url == 'undefined'
```

Comment:
Why does the first check use "triple equals" to check against `undefined`, but the second two checks only use a "double equals" check against `undefined` ?